util/test/builder/v1beta2_transition.go

```diff
@@ -206,11 +206,11 @@ type Phase2ObjStatus struct {
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
 
 	// +optional
-	Deprecated *Phase2ObjStatusDeprecated `json:"deprecated,omitempty"`
+	Deprecated *Phase2ObjDeprecatedStatus `json:"deprecated,omitempty"`
 }
 
-// Phase2ObjStatusDeprecated defines the status.Deprecated of a Phase2Obj.
-type Phase2ObjStatusDeprecated struct {
+// Phase2ObjDeprecatedStatus defines the status.Deprecated of a Phase2Obj.
+type Phase2ObjDeprecatedStatus struct {
 
 	// +optional
 	V1Beta1 *Phase2ObjStatusDeprecatedV1Beta1 `json:"v1beta1,omitempty"`
```

Comment:
Should we also rename Phase2ObjStatusDeprecatedV1Beta1 to Phase2ObjDeprecatedV1Beta1Status